### DISCUSSION TITLE
在仍有签到余额的情况下，是否充值$3就可以转为付费用户并保持下去？

### DISCUSSION THREAD

**Discussion by vivahu** [BurnHair/docs#163] [技术支持]
您好，首先感谢提供如此稳定&便宜的API。

通过查看FAQ得知免费和付费用户的QPS有比较大的差距。我目前靠每日签到能够维持日常使用（白嫖怪），但使用「沉浸式翻译」时确实会经常超时需要反复刷新。
<img width="694" alt="image" src="https://github.com/user-attachments/assets/b50abc5a-0e94-4399-9e40-54c011757994">

想问下：在仍有签到余额的情况下，是否充值$3就可以转为付费用户并保持下去？

感谢。。。

**Comment by uclinch** [comment]
是的

**Reply by lixiaodong007** [reply]
别上当了，以后付费用户也会清理

**Reply by BennyThink** [reply]
😆 https://github.com/orgs/BurnHair/discussions/176#discussioncomment-10217975
